openfisca_tasks/publish.mk

```diff
@@ -1,11 +1,11 @@
 .PHONY: build
 
-## Install openfisca-core for deployment and publishing.
+## Build & install openfisca-core for deployment and publishing.
 build:
 	@## This allows us to be sure tests are run against the packaged version
 	@## of openfisca-core, the same we put in the hands of users and reusers.
 	@$(call print_help,$@:)
-	@pip install --upgrade pip build twine
```

Comment:
Why do we remove the upgrade of `pip` and `twine` from here?

Reply:
Because they're not strictly needed for building.